python/pyspark/testing/__init__.py

```diff
@@ -14,3 +14,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+__all__ = ["assertSparkDFEquality", "assertDFEquality"]
```

Comment:
This PR exposes two API unlike how you described in PR description. Should probably either fix PR description or the change.